codemon/CodemonHelp.py

```diff
@@ -5,7 +5,6 @@ def showHelp():
   print(colored.green("           ---CODEMON---          "))
   print("A CLI tool to ace competitive programming contests")
   print(colored.cyan("\nCOMMANDS: \n"))
-  print("codemon - - - - - - - - - - - - - - -  shows help")
   print("codemon --help - - - - - - - - - - - - shows help")

```

Comment:
Change "show help" to "displays this message"

Reply:
Sure

Reply:
Done @ankingcodes